cmd/zetaclientd/start.go

```diff
@@ -221,6 +221,15 @@ func start(_ *cobra.Command, _ []string) error {
 		masterLogger.Info().Msg("TSS listener received an action to shutdown zetaclientd.")
 		signalChannel <- syscall.SIGTERM
 	})
+	// debug: printout connected peers
+	go func() {
```

Comment:
consider using `bg.Work()` with combination of `ticker.Run()`

Reply:
Agree

Reply:
not sure i understand it well to use it here. 
strict timing and dynamic adjustment is not needed here.